osc/commandline.py

```diff
@@ -4615,6 +4615,19 @@ def do_commit(self, subcmd, opts, *args):
             print("or use the --noservice option")
             return 1
 
+    def _has_pending_requests(self, api, prj, pkg, force=False):
+        if force:
+            return False
+
+        rqlist = get_request_list(api, prj, pkg)
+        if rqlist:
+            print("The following requests are pending for {}/{}:".format(prj, pkg))
```

Comment:
Last time I did this this returned incoming as well as outgoing requests. In this case this may trigger on a request that is still pending for factory.

Reply:
hm, this seems to depend on a _config value_.
uggghhhhhhhhh  why /bangs head against keyboard/